src/Fusion.Summary.Api/Deployment/k8s/deployment-pr-env.yml

```diff
@@ -0,0 +1,141 @@
+﻿#PS! 
+#secret pr.fusion-dev.net-tls must exist for this deployment to work. Since it contains secret information, it cannot be added to source control.
+
+apiVersion: v1
+kind: Secret
+metadata:
+  name: env-secrets
+  labels: 
+    environment: pr
+type: Opaque
+stringData:
+  appsettings.secrets.yaml: |
+    {
+      "AzureAd": {
+        "ClientSecret": "{{clientSecret}}"
+      }
+    }
+---
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: 'summary-api-pr-{{prNumber}}'
+  labels: 
+    environment: pr
+    prNumber: '{{prNumber}}'
+spec:
+  replicas: 1
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxSurge: 2
+      maxUnavailable: 50%
+  selector:
+    matchLabels:
+      app: 'summary-api-pr-{{prNumber}}'
+  template:
+    metadata:
+      labels:
+        app: 'summary-api-pr-{{prNumber}}'
+    spec:
+      volumes:
+      - name: env-secrets-volumes
+        secret: 
+          secretName: env-secrets
+
+      containers:
+      - name: summary-api-pr-{{prNumber}}
+        image: {{fusionAcr}}/{{imageName}}
+        imagePullPolicy: Always
+        ports:
+        - containerPort: 8080
+        volumeMounts:
+        - name: env-secrets-volumes
+          mountPath: "/app/secrets" 
+        env:
+        - name: APPINSIGHTS_INSTRUMENTATIONKEY
+          value: "{{instrumentationKey}}"
+        - name: KEYVAULT_URL
+          value: "{{envKeyVaultUrl}}"
+        - name: ENVNAME
+          value: "{{ENVNAME}}"
+        - name: FUSION_ENVIRONMENT
+          value: "{{fusionEnvironment}}"
+        - name: AzureAd__ClientId
+          value: "{{clientId}}"
+        - name: ConnectionStrings__ResourcesDbContext
```

Comment:
Is this something that we should remove for this pr?

Reply:
Fixd!